test/e2e/integration/payment-test/payment.ts

```diff
@@ -0,0 +1,105 @@
+import 'cypress-iframe';
+
+import { faker } from '@faker-js/faker';
+import path from 'path';
+
+import { AppFrontend } from 'test/e2e/pageobjects/app-frontend';
+const appFrontend = new AppFrontend();
+
+let paymentUrl: string;
+
+describe('Payment', () => {
+  beforeEach(() => {
+    cy.startAppInstance(appFrontend.apps.paymentTest, { authenticationLevel: '2' });
```

Comment:
Not sure how the authentication level will work on tt02, but it seems like the test app is not yet deployed there? :thinking: That should be working before we can merge this, otherwise we'll have constantly failing tests.